src/org/folio/rest_v2/Consortia.groovy

```diff
@@ -1,6 +1,8 @@
 package org.folio.rest_v2
 
+import org.folio.models.OkapiTenant
 import org.folio.models.OkapiTenantConsortia
+import org.folio.utilities.RestClient
```

Comment:
This 2 import is unnecessary please remove
import org.folio.models.OkapiTenant
import org.folio.utilities.RestClient